vignettes/LFQ_parse_qc_protein_inference.Rmd

```diff
@@ -367,103 +371,111 @@ Finally, we consider an alternative normalisation approach where we have a stron
 
 First, we need to calculate the RNase +/- ratios, since this is the value we want to normalise. In other applications, you may wish to normalise the abundance estimates for each sample, rather than the ratio between samples.
 ```{r}
-ratios <- exprs(prot_robust[,pData(prot_robust)$condition=='RNase_neg']) - exprs(prot_robust[,pData(prot_robust)$condition=='RNase_pos'])
+ratios <- exprs(prot_robust[, grepl("RNase_neg", colnames(prot_robust))]) - 
```

Comment:
Just to nit-pick, I prefer filtering using the `pData`, since this is the object explicit designed to hold information about the sample.

Reply:
Fair enough! I'll put it back next time I make some more commits 😉